hw04/BarbieWalletBalance.cs

```diff
@@ -5,17 +5,23 @@
 public class BarbieBank : MonoBehaviour
 {
 
-    // Start is called before the first frame update
+    BarbieWorld<string> barbieCareer;
 
     /* TODO: Problem 3: BARBIE'S BANK
     Convert the following function to a generic if needed. 
     Then, write a private generic function named BarbieBank. 
     BarbieBank should take in the parameters: numOfPennies, cashAmount, and numOfCreditCards
     Have the function return a new generic array with the correct parameters. 
     */
+    private T[] BarbieBank<T>(T numOfPennies, T cashAmount, T numOfCreditCards)
+    {
+        return new T[] { numOfPennies, cashAmount, numOfCreditCards };
+    }
 
     void Start()
     {
+        barbieCareer = new BarbieWorld<string>("Photographer");
```

Comment:
We will need to inherit from `BarbieWorld.cs` so we can also write it like this: 

```
GenericClass<int> myClass = new GenericClass<int>();
```

or for this problem:

```
BarbieWorld<string> barbieCareer = new BarbieWorld<string>("Photographer");
```